conf/uppmax.config

```
@@ -60,22 +62,32 @@ process {
   scratch = '$SNIC_TMP'
 }
 
-if (hostname.startsWith("s")) {
+// Cluster: Snowy
+//   Caution: Bianca nodes will be project name-nodenumber, e.g. sens2021500-001
+//   so cannot rely on just starting with 's'
+if (hostname ==~ "s[0-9][0-9]*") {
```

```suggestion
if (hostname.matches("^s[0-9][0-9]*")) {
```
Maybe use this if you don't want the `==~`